src/scc_hypervisor_collector/api/uploader.py

```diff
@@ -5,7 +5,59 @@
 collected from the specified backends to the SCC using the provided
 credentials.
 """
+import json
+import logging
+import gzip
+from typing import Dict
+from importlib_metadata import version
+import requests
+from requests.exceptions import RequestException
+from .configuration import SccCredsConfig
 
 
 class SCCUploader:
     """SCC Uploader for scc-hypervisor-collector."""
+
+    def __init__(self, scccredsconfig: SccCredsConfig):
```

Comment:
I would probably just call the parameter `scc_creds` - no need to include the `config` in the parameter name, and use the `_` to separate the `c`'s.